TodoApp/Tests/TodoApp.TestContracts/Wrappers/ArgWrapper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using NSubstitute;
+using TodoApp.Contracts.Models;
+using TodoApp.TestContracts.Extensions;
+
+namespace TodoApp.TestContracts.Wrappers
+{
+    public static class ArgWrapper
+    {
+        private static readonly Lazy<ItemComparer> ItemComparer = new Lazy<ItemComparer>();
```

Comment:
Move this field to the `ItemComparer` class itself and privatize its constructor so only one instance of the `ItemComparer` appears in your tests - after all, it is quite a solid candidate for a singleton, isn't it? 

Introducing a "non-lazy" property that would always return the value of the lazy backing field would be most welcomed since it hides singleton's implementation details.